<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SongPlayerUi</class>
 <widget class="QWidget" name="SongPlayerUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="OpenButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PlayButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>230</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
  </widget>
  <widget class="QLabel" name="infoLabel">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>110</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextSong">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>150</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>-&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="prevSong">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>80</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>241</width>
     <height>22</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
